libraries/AP_NavEKF2/AP_NavEKF2_Outputs.cpp

```diff
@@ -594,10 +594,10 @@ void NavEKF2_core::send_status_report(mavlink_channel_t chan)
 // report the reason for why the backend is refusing to initialise
 const char *NavEKF2_core::prearm_failure_reason(void) const
 {
-    if (imuSampleTime_ms - lastGpsVelFail_ms > 10000) {
-        // we are not failing
-        return nullptr;
-    }
+//    if (imuSampleTime_ms - lastGpsVelFail_ms > 10000) {
+//        // we are not failing
+//        return nullptr;
+//    }
```

Comment:
Why is this commented?

Reply:
as stated in the description:
"I've just commented out the code which was suppressing that error report; perhaps it would be better instead to enforce the appropriate EKn_ENABLE to match AHRS_EKF_TYPE ?"